### ISSUE TITLE
indicator kde-connect not working

### ISSUE THREAD

**Issue by ghost** [vikoadi/indicator-kdeconnect#38]
ayush@ayush-VPCEB44EN:~$ indicator-kdeconnect 
ayush@ayush-VPCEB44EN:~$ 

I am using Ubuntu 14.04 and indicator kdeconnect application just wont launch .

**Comment by spravo13** [comment]
I am having the same problem with debian testing

I recieve an alert with the message: "cannot connect to KDE Connect DBus service"

**Comment by diegonz** [comment]
Same behaviour on Ubuntu 15.10, pairing and receiving notifications on pc works great if done from kdeconnect-plasma.
Also indicator wizard runs good, but the icon(s) doesn´t show at all

**Comment by armc** [comment]
Is anyone working this issue??.  It would be great to get a working package back in circulation.

**Comment by spravo13** [comment]
I agree, I've been swamped so I haven't had a chance to look into the problem any further than before. Please update this thread if you find any additional information. Just got on Spring Break today (Wooo!!) so I will let you guy's know through this with what I find.

**Comment by armc** [comment]
Excellent.  Good luck... but enjoy your spring break first and foremost!

**Comment by ghost** [comment]
Try this
open terminal try use this command "kcmshell4 kcm_kdeconnect"
and try to remove the device or unpair and then try it.

**Comment by armc** [comment]
I will try this when I get the chance... been packing for a trip.  I'll post an update to my experience with the kcmshell4 kcm_kdeconnect and then unpair / repair.  No time presently.  Thank!!! :)

**Comment by Natsumi-sama** [comment]
I'm getting the error:
*\* Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/d2d4bb6f54282ffa' (signature '')

**Comment by 1595ayush** [comment]
@cmcooper could you be more specific . What command did you run ?

**Comment by ghost** [comment]
@cmcooper123 couldn't you be more specific . What command did you run ?

**Comment by aldomann** [comment]
@1595ayush I get the same message when launching `indicator-kdeconnect` from the Terminal, so it must be it. To be more specific, I get this:

```
** Message: KDEConnectManager.vala:32: KDE Connect daemon found
** Message: Device.vala:73: device : /modules/kdeconnect/devices/e1342f8007f52a77
** Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/e1342f8007f52a77' (signature '')
** Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/e1342f8007f52a77' (signature '')
** Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/e1342f8007f52a77' (signature '')
** Message: Device.vala:263: GDBus.Error:org.freedesktop.DBus.Error.UnknownMethod: No such method 'isReachable' in interface 'org.kde.kdeconnect.device' at object path '/modules/kdeconnect/devices/e1342f8007f52a77' (signature '')
```

**Comment by ghost** [comment]
Well you can either use plasma(see 3rd comment by btsunami) or go to issue number #31 @aldomann

**Comment by Natsumi-sama** [comment]
@1595ayush indicator-kdeconnect, I tried removing ~/.config/kdeconnect and running "kcmshell5 kcm_kdeconnect" before after and during running indicator-kdeconnect

**Comment by lutostag** [comment]
With a new build from https://github.com/rcristi/indicator-kdeconnect I got it working for me. I was running into the same issue.
PPA here: https://launchpad.net/~lutostag/+archive/ubuntu/indicator-kdeconnect-test

(couldn't get it to build for precise with the versioned dependencies). Not sure if they are necessary of not.

**Comment by NiklasMerz** [comment]
I have the same problem after upgrading my Linux Mint 17.3 (Ubuntu 14.04) to Mint 18 (Ubuntu 16.04)

**Comment by pandacriss** [comment]
lutostag, i compile your version and it work. very thankful.

**Comment by Simon0Harms** [comment]
Same Problem here with Ubuntu Mate 16.04 64 Bit

**Comment by Simon0Harms** [comment]
@lutostag Your version don't work for me on Ubuntu Mate 16.04 64 Bit, too
 *\* Message: KDEConnectManager.vala:29: retrying to find KDE Connect DBus service
*\* Message: KDEConnectManager.vala:29: retrying to find KDE Connect DBus service
*\* Message: KDEConnectManager.vala:29: retrying to find KDE Connect DBus service
*\* Message: KDEConnectManager.vala:29: retrying to find KDE Connect DBus service
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.

**Comment by lupa18** [comment]
Not working here. Ubuntu 16.04

**Comment by ghost** [comment]
I am also having same issue in Ubuntu 16.04 !   indicator popups do not work. Only the ping showed when i pinged from phone as soon as i installed everything on Ubuntu 16.04. 
then i restarted the pc and none of the popup indicators work. the only notifications i get from my phone to my pc is from the kde-monitor.  

Also the kde-monitor i cannot see a mirror of my phone so i cannot remote control.  I only see battery and notifications, and when i close kde-monitor, and i re-open it, it doesnt open again until i kill the process manually, then i can re-open it. 

kde-monitor has no menu or indicator taskbar settings to be able to minimize it in taskbar, or close in taskbar.

Please can someone try fix the indicator problems on ubuntu 16.04?

**Comment by Beast141** [comment]
i get this when i type

**Comment by ghost** [comment]
i get this when i type:
kcmshell4 kcm_kdeconnect

Connecting to deprecated signal QDBusConnectionInterface::serviceOwnerChanged(QString,QString,QString)
QDBusConnection: session D-Bus connection created before QCoreApplication. Application may misbehave.
QDBusConnection: session D-Bus connection created before QCoreApplication. Application may misbehave.
kbuildsycoca4 running...
kbuildsycoca4(5514) KBuildSycoca::checkTimestamps: checking file timestamps
kbuildsycoca4(5514) KBuildSycoca::checkTimestamps: timestamps check ok
kbuildsycoca4(5514) kdemain: Emitting notifyDatabaseChanged ()
kcmshell(5483)/kdecore (services) KServiceFactory::findServiceByDesktopPath: "kcm_kdeconnect.desktop" not found
kcmshell(5483)/kdecore (services) KServiceFactory::findServiceByDesktopPath: "kcm_kcm_kdeconnect.desktop" not found
kcmshell(5483)/kdecore (services) KServiceFactory::findServiceByDesktopPath: "kcmkcm_kdeconnect.desktop" not found
Could not find module 'kcm_kdeconnect'. See kcmshell4 --list for the full list of modules.

**Comment by NeoMahler** [comment]
same as @Beast141

**Comment by Tehmal** [comment]
reflesh

indicator-kdeconnect 
** Message: KDEConnectManager.vala:29: retrying to find KDE Connect DBus service
** Message: KDEConnectManager.vala:29: retrying to find KDE Connect DBus service
** Message: KDEConnectManager.vala:29: retrying to find KDE Connect DBus service
** Message: KDEConnectManager.vala:29: retrying to find KDE Connect DBus service
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.
Naruszenie ochrony pamięci (Core Dump)

**Comment by sollidsnake** [comment]
I tried `kcmshell5 kcm_kdeconnect` instead of `kcmshell4` and it worked. Knowing this is there a way to run the indicator?

**Comment by Simon0Harms** [comment]
This one is working:
https://github.com/Bajoja/indicator-kdeconnect
